template_config.yml

```diff
@@ -48,5 +60,7 @@ test_s3: false
 travis_addtl_services: []
 travis_notifications: None
 update_redmine: false
-upgrade_range: []
+upgrade_range:
+- pulp_python_branch: 3.2
```

Comment:
@gerrod3 @dralley could you please create this branch?

Reply:
Branch should be created now. Do you also need a 3.3 branch?

Reply:
no, just the 3.2, I'll be adding the pre_upgrade tests there